### ISSUE TITLE
how to install sqlcipher

### ISSUE THREAD

**Issue by alicera** [sqlitebrowser/sqlitebrowser#2518]
OS: ubuntu18.04
kernel: 5.6

I install the sqlitebrowser and open the DB.
How to open it with sqlcipher?

![image](https://user-images.githubusercontent.com/59388989/103058591-82d4b880-45dd-11eb-9b15-335c72f4cc83.png)

**Comment by scottfurry** [comment]
For Ubuntu, I believe you need sqlcipher package(s) installed - sqlcipher-dev, sqlcipher and/or libsqlcipher0.
Package search can be checked here: https://packages.ubuntu.com/

**Comment by alicera** [comment]
I follow https://stackoverflow.com/questions/48105035/sqlite-browser-without-sqlcipher-support-in-ubuntu
and open DB
PS the method have the install step of sqlcipher package(s) 

Then I open DB for security and input the password
![image](https://user-images.githubusercontent.com/59388989/103061974-c8967e80-45e7-11eb-86a9-6d9b9fb57843.png)
![image](https://user-images.githubusercontent.com/59388989/103062075-2925bb80-45e8-11eb-92f5-ba7038af1e73.png)

I make sure the DB can be open for Windows
![image](https://user-images.githubusercontent.com/59388989/103062192-74d86500-45e8-11eb-9807-71d28074f7d8.png)

**Comment by justinclift** [comment]
Yeah.  When the application is built, it needs to be compiled with either standard SQLite **or** SQLCipher.  It's one or the other.

You can check which one it's compiled for, but going to Help -> About in the main menu.  That should give some information about the version of SQLite or SQLCipher it was compiled with.

**Comment by justinclift** [comment]
Also, do you know what version of SQLCipher was used for creating your database file?  Asking because SQLCipher 3.x database files are a bit different to SQLCipher 4.x ones.  There is an option in the password dialog that lets you choose which one you're opening (`Encryption settings`).  So you might need to try some of the options there too. :smile:

As a data point, do if the encrypted database you're trying to open

**Comment by alicera** [comment]
I use the https://github.com/sqlcipher/sqlcipher (4.4.2)  to generate the DB 

This is my version
![image](https://user-images.githubusercontent.com/59388989/103085438-a0754280-461c-11eb-8901-0a45948d1c8d.png)


 I already try some of the options for sqlcipher3 and sqlcipher4.
But it still fail.
> <

**Comment by chrisjlocke** [comment]
I would assume (although its just a guess) that if you've created a sqlCipher v4 database, it cannot be opened using sqlCipher v3.
The version of sqlCipher we're using (at least with the Windows builds) is 4.4

**Comment by chrisjlocke** [comment]
Aah yes - see here : https://discuss.zetetic.net/t/can-i-open-a-v4-db-file-using-sqlcipherv3/3401
The v3 library (which you're using) cannot open a v4 database (which you've created).

**Comment by alicera** [comment]
how to install v4?
I follow the step  https://stackoverflow.com/questions/48105035/sqlite-browser-without-sqlcipher-support-in-ubuntu .
It will be v3

**Comment by chrisjlocke** [comment]
I don't use Ubuntu, so can't advise.  That post is three years old though, so that would explain why it's referring to old packages.
Hopefully someone more knowledgeable than me can help though... we have a fair few Linux users floating around.

**Comment by alicera** [comment]
Hopefully someone more knowledgeable can help!
![image](https://user-images.githubusercontent.com/59388989/103623058-84ee2e00-4f72-11eb-98d4-e1a2534a7bc5.png)
